functions/tide/configure/configs/classic_16color.fish

```diff
@@ -10,6 +10,7 @@ tide_context_color_root bryellow
 tide_context_color_ssh yellow
 tide_docker_bg_color black
 tide_docker_color blue
+tide_docker_default_contexts default colima
```

Comment:
This variable shouldn't be in any of the 16_color configs. Those configs are for modifying colors only, they inherit all of the data variables.

Reply:
OK. I’ll adjust that in a few hours and push up a new version.

Reply:
Sorry. Just resolved this.